README.rst

```diff
@@ -380,6 +382,33 @@ Remove a Resource
     # delete a resource
     >>> lun1.delete()
 
+Getting metrics
+```````````````
+
```

Comment:
Could you add all supported metrics from metric_configs.yaml? Just list the names.

Reply:
Done.